source/whats-new.txt

```diff
@@ -1 +1,18 @@
-TODO: write
+.. _mongodb-analyzer-changelog:
+
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+What's New in 1.0
+-----------------
+
+The initial GA release.
```

Comment:
Please expand "GA" to general availability or omit it. It's not a term that everyone uses nor is familiar with.

Reply:
Removed "GA". Now it's just "The initial release."